contracts/BullaFactoring.sol

```diff
@@ -297,12 +306,13 @@ contract BullaFactoringV2_1 is IBullaFactoringV2, ERC20, ERC4626, Ownable {
             feeParams: feeParams,
             fundedAmountGross: 0,
             fundedAmountNet: 0,
-            initialInvoiceValue: _initialInvoiceValueOverride != 0 ? _initialInvoiceValueOverride : invoiceSnapshot.invoiceAmount - invoiceSnapshot.paidAmount,
+            initialInvoiceValue: _initialInvoiceValue,
             initialPaidAmount: invoiceSnapshot.paidAmount,
             receiverAddress: address(0),
             invoiceDueDate: invoiceSnapshot.dueDate,
             impairmentDate: invoiceSnapshot.dueDate + invoiceSnapshot.impairmentGracePeriod,
-            protocolFee: 0
+            protocolFee: 0,
+            dailyInterestRate: Math.mulDiv(_initialInvoiceValue, _targetYieldBps, 3_650_000)
```

Comment:
is the calculation here necessary? it gets overriden once the actual funding takes place